docs/content/concepts.mdx

```diff
@@ -350,12 +350,27 @@ This will affect only relations that are [directly related](#what-are-direct-and
 
 </details>
 
+<details>
+<summary>
+## What is an ABAC Condition?
+
+An **ABAC Condition** is a function composed of one or more parameters and an expression which must evaluate to a boolean outcome. Expressions are defined using [Google's Common Expression Language (CEL)](https://github.com/google/cel-spec).
+</summary>
+
+For example, the following condition `mycondition` defines an ABAC Condition which evaluates to a boolean outcome. The provided parameter `x`, defined as an integer type, is used in the boolean expression `x < 100`. The condition will return a truthy outcome if the expression returns a truthy outcome and false otherwise.
```

Comment:
```suggestion
For example, in the following snippet `mycondition` defines defines a Condition which evaluates to a boolean outcome. The provided parameter `x`, defined as an integer type, is used in the boolean expression `x < 100`. The condition will return a truthy outcome if the expression returns a truthy outcome and false otherwise.
```

Reply:
Can we use something other than `mycondition` as a name? `less_than_hundred` for example